app/src/hooks/AuthHandler.ts

```diff
@@ -68,6 +68,10 @@ const AuthHandler: React.FC<{}> = () => {
 
       const userData = await getUser(user.uid);
 
+      if (userData?.browserstackId) {
+        submitAttrUtil(TRACKING.ATTR.BROWSERSTACK_ID, userData.browserstackId);
```

Comment:
When `userData?.browserstackId` doesn't exist, we should reset it back to null to handle logout or user change cases

Reply:
I think null is rejected in our submitAttrUtil method. We might need to fix that too